Duplicati/Library/Main/Controller.cs

```diff
@@ -288,14 +288,12 @@ public Duplicati.Library.Interface.IBackupResults Backup(string[] inputsources,
                 }
 
                 //Sanity check for duplicate files/folders
-                var pathDuplicates = sources.GroupBy(x => x, Library.Utility.Utility.ClientFilenameStringComparer)
-                      .Where(g => g.Count() > 1).Select(y => y.Key).ToList();
+                ISet<string> pathDuplicates;
+                sources = Library.Utility.Utility.GetUniqueItems(sources, Library.Utility.Utility.ClientFilenameStringComparer, out pathDuplicates).OrderBy(a => a).ToList();

```

Comment:
Is the `OrderBy` necessary?  I included it because the original implementation sorted the results.

Reply:
I think it is used later to find out if an included folder is a sub-folder of a top-level folder.